xsd/netex_part_1/part1_networkDescription/netex_routeInstruction_version.xsd

```diff
@@ -73,11 +73,18 @@ Rail transport, Roads and Road transport
 		<xsd:complexContent>
 			<xsd:extension base="strictContainmentAggregationStructure">
 				<xsd:sequence>
-					<xsd:element ref="RouteInstruction" maxOccurs="unbounded">
-						<xsd:annotation>
-							<xsd:documentation>An Instruction on how to follow a ROUTE through the network.</xsd:documentation>
-						</xsd:annotation>
-					</xsd:element>
+					<xsd:choice>
+						<xsd:element ref="RouteInstructionRef" maxOccurs="unbounded">
```

Comment:
Would it be possible to do this in the following way:
<choice>
  <sequence>
    <xsd:element ref="RouteInstructionRef" maxOccurs="unbounded">
  </sequence>
  <sequence>
    <xsd:element ref="RouteInstruction" maxOccurs="unbounded">
  </sequence>
</choice>

Reply:
it is... but would be a bit strange. Do you need it for the conversion? commit done

Reply:
I would love to have a conversation on this being our 'default' way to do containers with aggregations, so that its either a List[Type1] or List[Type2]. Sure in ServiceJourneyPattern it serves a function, I cannot wrap my hand around why you want to instantiate a type here (being the RouteInstruction) and then next in the list you make it a RouteInstructionRef.